pkgs/applications/misc/watershot/default.nix

```
@@ -0,0 +1,31 @@
+{ lib, rustPlatform, fetchFromGitHub, pkg-config, wayland, libxkbcommon
+, fontconfig, makeWrapper, grim }:
+rustPlatform.buildRustPackage rec {
+  pname = "watershot";
+  version = "0.1.2";
+
+  src = fetchFromGitHub {
+    owner = "Kirottu";
+    repo = "watershot";
+    rev = "v${version}";
+    hash = "sha256-8GqO7Y0d+AoYr3Us3FEfNobrQNSw7XyGwmZz5HkVvDg=";
+  };
+
+  cargoLock.lockFile = src + "/Cargo.lock";
+
+  nativeBuildInputs = [ pkg-config wayland makeWrapper ];
+
+  buildInputs = [ wayland fontconfig libxkbcommon ];
+
+  postInstall = ''
+    wrapProgram $out/bin/watershot \
+      --prefix PATH : ${lib.makeBinPath [ grim ]}
+  '';
+
+  meta = with lib; {
+    description = "A simple wayland native screenshot tool.";
```

```suggestion
    description = "A simple wayland native screenshot tool";
```

`meta.description` should not end with a period.
See https://nixos.org/manual/nixpkgs/unstable/#var-meta-description and https://github.com/NixOS/nixpkgs/blob/11eb1c2bc981ae4279b5d06044ad38ec855193e4/CONTRIBUTING.md#submitting-changes